### DISCUSSION TITLE
Avatar images not displaying

### DISCUSSION THREAD

**Discussion by christoferd** [thedevdojo/wave#163] [General]
Issue: Avatar images not displaying
Both, the default admin avatar is not being displayed, and another avatar that I uploaded.

Note: upload avatar image in user create form did work OK and file stored in local storage.

Platform: Windows Herd auto install Wave v3 (following the install guide)

![image](https://github.com/user-attachments/assets/365af88d-fe3e-4f93-bbc4-63e6193c2d41)

Default file system config
![image](https://github.com/user-attachments/assets/8af3885a-a09a-4c49-b235-e20bd88f33ca)

Admin avatar file appears to be in the correct location
![image](https://github.com/user-attachments/assets/96091288-5979-4111-986e-7237e9f2eed3)

**Comment by bobbyiliev** [comment]
Hi,

I have not been able to reproduce this on my end:

<img width="177" alt="image" src="https://github.com/user-attachments/assets/15d63be3-72fd-4b5f-ad49-055cbca400f0">

I tried both the admin and the `/settings/profile`.

The default one also seems to be loading as expected:

<img width="1451" alt="image" src="https://github.com/user-attachments/assets/e71959ce-ca42-49b9-a481-da743e9ebfd1">

Can you confirm that you have your storage link setup correctly?

Also which theme are you using?

**Comment by louisfilhol** [comment]
same issue here


![image](https://github.com/user-attachments/assets/183e2f16-b10a-4699-9367-3c885f0cd445)
![image](https://github.com/user-attachments/assets/796f87e5-63c7-442e-b127-1bbff2a455c3)

**Reply by bobbyiliev** [reply]
I've not been able to reproduce this. On which page do you see that exactly? How did you upload the logo, via Filament or via the profile? Which theme are you using?

**Comment by eloquentbit** [comment]
I had the same issue on a fresh install; in my case the problem was the APP_URL variable in the .env file which didn't reflect the correct domain name: <app_name>.test (I use Herd on macOS).

**Reply by bobbyiliev** [reply]
Ah yes, this is indeed very important! Thanks for pointing it out!
